demo/expressServer.js

```diff
@@ -1,18 +1,35 @@
-var express = require('express');
-var http = require('http');
-var serveIndex = require('serve-index');
+
+import {globPatterns} from '../test/modular/NodeTraceurTestRunner.js';
+
+let express = require('express');
+let http = require('http');
+let serveIndex = require('serve-index');
 
 function servePathAtPort(path, port) {
-  var app = express();
-  app.use(express.static(path));   // before directory to allow index.html to work
+  let app = express();
+  // serveIndex must precede static to allow index.html to work
   app.use(serveIndex(path));
-  var server = http.createServer(app);
+  app.use(express.static(path));
+  // Expand the test list based on the file system.
+  app.get('/traceurService/testGlobs', function(req, res) {
+    let patterns = JSON.parse(req.query.patterns);
+    return globPatterns(patterns).then((files) => {
+      let nodeless = [];
+      files.forEach((file) => {
+        if (file.indexOf('/node/') === -1) {
+          nodeless.push(file);
+        }
+      });
+      return res.send(nodeless);
+    });
+  });
+  let server = http.createServer(app);
   server.on('error', function(e) {
-  	console.log('Port ' + port + ' did not work out');
+    console.error('Port ' + port + ' did not work out');
   });
   server.listen.apply(server, [port]);
   console.log('serving ' + path + ' at ' + port);
 }
 
-servePathAtPort(__dirname + '/..', 8099);
-servePathAtPort(__dirname + '/..', 80);
+servePathAtPort(System.dirname(__moduleName) + '/..', 8099);
```

Comment:
https://github.com/whatwg/loader/issues/36
search for 'async import'.  As far as I can tell, this is about correctly setting the referrer for import done by function call.  That much is dealt with by node using a function wrapper that passes in a version of `require()` that knows the callers module path.  That same wrapper in node sets the `__dirName` value for the module, so I'm extrapolating to imagine that the same solution would apply.

To be honest, I think such experimenting would be mostly for furn.  I think node will just continue on with the require wrapper thing.